src/bicep-types-go/Makefile

```diff
@@ -0,0 +1,99 @@
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+GOMOD=$(GOCMD) mod
+GOFMT=$(GOCMD) fmt
+
+# Build parameters
+BINARY_NAME=bicep-types
+BINARY_PATH=./cmd/bicep-types
+BUILD_DIR=./bin
+
+# Test parameters
+TEST_PACKAGES=./...
+TEST_FLAGS=-v -race -cover
+
+.PHONY: all build test clean fmt vet tidy deps help
+
+# Default target
+all: clean fmt vet test build
+
+# Build the binary
+build:
+	@echo "Building $(BINARY_NAME)..."
+	@mkdir -p $(BUILD_DIR)
+	$(GOBUILD) -o $(BUILD_DIR)/$(BINARY_NAME) $(BINARY_PATH)
+	@echo "Built $(BUILD_DIR)/$(BINARY_NAME)"
+
+# Run all tests
+test:
+	@echo "Running tests..."
+	$(GOTEST) $(TEST_FLAGS) $(TEST_PACKAGES)
+
+# Run tests with coverage report
+test-coverage:
+	@echo "Running tests with coverage..."
+	$(GOTEST) -v -race -coverprofile=coverage.out $(TEST_PACKAGES)
+	$(GOCMD) tool cover -html=coverage.out -o coverage.html
+	@echo "Coverage report generated: coverage.html"
+
+# Run specific test
+test-verbose:
+	@echo "Running tests with verbose output..."
+	$(GOTEST) -v $(TEST_PACKAGES)
+
+# Clean build artifacts
+clean:
+	@echo "Cleaning..."
+	$(GOCLEAN)
+	@rm -rf $(BUILD_DIR)
+	@rm -f coverage.out coverage.html
+
+# Format Go code
+fmt:
+	@echo "Formatting code..."
+	$(GOFMT) ./...
+
+# Run linter (requires golangci-lint)
+lint:
+	@echo "Running linter..."
+	@which golangci-lint > /dev/null || (echo "golangci-lint not found, install with: go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest" && exit 1)
```

Comment:
Is it possible to configure this linter to verify the correct copyright header has been added? This is how we have our TS and C# projects configured - it's one less thing to have to remember when reviewing code.